config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -94,6 +94,7 @@ periodics:
 
               TIMESTAMP=$(date +%s)
               K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/kubernetes-release-dev/ci/latest.txt)
+              jq --arg key0 'k8s-build-version' --arg value0 $K8S-BUILD-VERSION '. | .[$key0]=$value0' <<<'{}' > $ARTIFACTS/metadata.json
```

Comment:
I see typo in the variable name used! `$K8S-BUILD-VERSION`, it should be `$K8S_BUILD_VERSION`

Reply:
That was in hurry while I replaced underscore with hypen in the key :(